src/main/java/net/glowstone/block/GlowBlock.java

```diff
@@ -452,5 +461,6 @@ public void applyPhysics(Material oldType, int newTypeId, byte oldData, byte new
         if (type != null) {
             type.onBlockChanged(this, oldType, oldData, newType, newData);
         }
+        type.receivePulse(this);
```

Comment:
Well `.update()` with no applyPhysics still tries to update the block, so we should still pulse the block when it is updated.